### ISSUE TITLE
1.8.0 problem

### ISSUE THREAD

**Issue by pladaria** [prettier/prettier-vscode#705]
Latest version (1.8.0) is not working for me. Previous version (1.7.3) working fine.

```
$ code --version
1.30.2
61122f88f0bf01e2ac16bdb9e1bc4571755f5bd8
x64

$ uname -a
Linux xtc 4.10.0-40-generic #44~16.04.1-Ubuntu SMP Thu Nov 9 15:37:44 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

Error message
```
31/1/2019 17:28:39:
-------------------
Couldn't resolve parser "babel"
```

.prettierrc
```
{
    "printWidth": 110,
    "tabWidth": 4,
    "singleQuote": true,
    "trailingComma": "es5",
    "bracketSpacing": false,
    "proseWrap": "always",
    "overrides": [
        {
            "files": "package.json",
            "options": {
                "printWidth": 250
            }
        },
        {
            "files": "*.md",
            "options": {
                "tabWidth": 2
            }
        }
    ]
}
```

First lines of .eslintrc (not sure if relevant)
```
root: true
parser: babel-eslint
settings:
    react:
    ...
```

**Comment by atabel** [comment]
Same here.
Seems to work in Mac, but not in Linux

**Comment by francocorreasosa** [comment]
Same, had to downgrade on Mac.

**Comment by glebec** [comment]
Had to downgrade to an earlier version (on MacOS 10.4), using the following steps:

I have figured out a temporary workaround:

1. Download `https://marketplace.visualstudio.com/_apis/public/gallery/publishers/esbenp/vsextensions/prettier-vscode/1.8.0/vspackage`
2. In VSC, uninstall the Prettier extension
3. `CMD + SHIFT + P` and type `VISX` + `ENTER` to select "…install from VISX"
4. Select the downloaded file
5. Reload VSC when it tells you to
6. Voilà, it works again.

Note, you may or may not have to disable extension auto-updates to prevent the regression from coming back. I haven't checked that yet.

**Comment by andrewtpoe** [comment]
1.8.0 is broken for me as well on mac. Same error.

**Comment by CiGit** [comment]
`getSupportInfo` seems to lie :-)

A current workaround could be (I didn't tested it):
a prettier config file with
```json
{ 
"overrides": [
    {
      "files": "*.js",
      "options": {
        "parser": "babylon"
      }
    }
  ]
}
```

**Comment by CiGit** [comment]
Fixed in #706. Released in 1.8.1
Thanks @JHilker !
